src/main/java/core/basesyntax/HelloWorld.java

```diff
@@ -1,8 +1,26 @@
 package core.basesyntax;
 
-/**
- * Feel free to remove this class and create your own.
- */
-public class HelloWorld {
+import java.util.Random;
+
+public class HelloWorld extends CircleSupplier {
+    public static void main(String[] args) {
+        final Random random = new Random();
+        Circle circle = new Circle();
+        IsoscelesTrapezoid isoscelesTrapezoid = new IsoscelesTrapezoid();
+        Rectangle rectangle = new Rectangle();
+        RightTriangle rightTriangle = new RightTriangle();
+        Square square = new Square();
+        Figure[] choise = {circle, isoscelesTrapezoid, rectangle, rightTriangle, square};
+        Figure[] figures = new Figure[6];
+        for (int i = 0; i < figures.length / 2; i++) {
+            figures[i] = choise[random.nextInt(choise.length)];
+            figures[i].getArea();
+            figures[i].draw();
+        }
+
+        for (int i = figures.length / 2; i < figures.length; i++) {
+            new CircleSupplier().writeDefaultCircle();
```

Comment:
`Figure createDefaultFigure()` could be a private method of this class.

Reply:
![image](https://github.com/mate-academy/jv-oop-advanced/assets/68182494/4b7bed45-dbd6-4292-afd3-a95e7e54e187)
![image](https://github.com/mate-academy/jv-oop-advanced/assets/68182494/da444b1f-8051-4f15-a77d-f45bf5250a32)
may it be like this? cuz i thought that in README was said about creating the method in parent class Figure

Reply:
Please read carefully - you're supposed to create this method in `FigureSupplier` class, not `Figure` itself.